### PR TITLE
DOC: change link to contributing guide in README.md

### PR DESCRIPTION
**Pull request by simonjayhawkins**
xref https://github.com/pandas-dev/pandas/pull/32068#issuecomment-593362224

### CODE REVIEW

**Comment by jbrockmendel** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ Most development discussion is taking place on github in this repo. Further, the
 
 All contributions, bug reports, bug fixes, documentation improvements, enhancements and ideas are welcome.
 
-A detailed overview on how to contribute can be found in the **[contributing guide](https://dev.pandas.io/docs/contributing.html)**. There is also an [overview](.github/CONTRIBUTING.md) on GitHub.
+A detailed overview on how to contribute can be found in the **[contributing guide](https://pandas.io/docs/development/contributing.html)**. There is also an [overview](.github/CONTRIBUTING.md) on GitHub.
```
Comment (jorisvandenbossche) [inline README.md]: Can you leave the `dev.` for now? (until we have a proper place for the dev docs) The link you are changing it to now are the stable docs

Comment (simonjayhawkins) [reply README.md]: > The link you are changing it to now are the stable docs

see https://github.com/pandas-dev/pandas/pull/32068#issuecomment-593362224

dev docs are stale

image


this new link is not the stable docs

image

**Comment by datapythonista** [comment]
Working on the actual fix now, see #32530

**Comment by simonjayhawkins** [comment]
Thanks @datapythonista

**Comment by simonjayhawkins** [comment]
> Can you leave the dev. for now? (until we have a proper place for the dev docs) The link you are changing it to now are the stable docs

updated.

**Review by datapythonista — APPROVED** [review]
lgtm, thanks Simon

**Pull request merged**